scripts/all-tests.sh

```diff
@@ -126,8 +93,22 @@ END
         ;;
 
     "unit")
-        paver test
-        paver coverage
+        case "$SHARD" in        
+            "lms")
+                paver test_system -s lms
+                paver coverage
+                ;;
+            "other")
```

Comment:
I know it's not as pretty, but maybe we call this something more descriptive, like cms_js_common